sap-labs-france/ev-server
# Asset integration test cases for ioThink

Test cases for ioThink asset integration
Before executing the test case, set the below parameters in the local.json
```
"assetConnectors": {
  "ioThink": {
    "user": ioThink username,
    "password": ioThink password,
    "meterID": meterID connection string used to connect to actual asset,
    "url": optional parameter, defaults to "https://api.kheiron-sp.io"
  }
}
```

Test:
- Setup the parameters as metioned above
- Verify if the asset connector test cases for ioThink are passing successfully
